modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -3511,6 +3513,30 @@ private CacheConfiguration getOrCreateConfigFromTemplate(String cacheName) throw
         }
     }
 
+    /**
+     * Prepares encryption key for the cache or return {@code null}
+     * if it will use a key with stored on metasatorage.
```

Comment:
key with stored on metasatorage. -> key which is stored in the metasatorage.

Reply:
Done.